mindmaps-engine/src/main/java/io/mindmaps/api/ImportController.java

```diff
@@ -132,6 +133,7 @@ void importDataFromFile(String dataFile) throws IOException {
         loader.waitToFinish();
         QueryParser.create().parsePatternsStream(new FileInputStream(dataFile)).forEach(pattern -> consumeRelation(pattern.admin().asVar()));
         loader.waitToFinish();
+        BackgroundTasks.getInstance().forcePostprocessing();
```

Comment:
This means that after each data import from a file we run BackgroundTasks ? Could this accidentally result in 2 or more BackgroundTasks being run ?